.github/workflows/wheel.yaml

```diff
@@ -6,9 +6,21 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+env:
+  # Build all wheels on either:
+  #    - tagged release
+  #    - pull request opened with "test all wheels" label
+  #    - pull requests when "test all wheels" label is added
+  BUILD_ALL: |
+    ${{
+        (github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/v'))
+        || contains(github.event.pull_request.labels.*.name, 'test all wheels')
+        || (github.event.action == 'labeled' && github.event.label.name == 'test all wheels')
+    }}
```

Comment:
Ok so in this case we are relying on maintainers to add the label for the wheel builds

Is it worth considering an option users can opt-in with? Like commit message text or PR title?